.github/renovate.json

```diff
@@ -2,19 +2,34 @@
   "automerge": true,
   "enabledManagers": ["github-actions", "npm"],
   "ignoreDeps": ["cypress"],
-  "internalChecksFilters": "strict",
+  "internalChecksFilter": "strict",
   "labels": ["dependencies"],
   "packageRules": [
     {
-      "addLabels": ["github-actions"],
-      "manager": "github-actions"
+      "matchManagers": ["npm"],
+      "addLabels": ["javascript"]
     },
     {
-      "addLabels": ["javascript"],
-      "manager": "npm"
+      "matchManagers": ["github-actions"],
+      "addLabels": ["github-actions"]
+    },
+    {
+      "matchUpdateTypes": ["major"],
+      "addLabels": ["major"]
+    },
+    {
+      "matchUpdateTypes": ["pin"],
+      "addLabels": ["pin"]
+    },
+    {
+      "matchUpdateTypes": ["patch"],
+      "addLabels": ["patch"]
+    },
+    {
+      "matchUpdateTypes": ["digest"],
+      "addLabels": ["digest"]
     }
```

Comment:
We don't do this today -and didn't filter on version upgrade type previously- so I feel like it's fine to just ignore them now.

```suggestion
```

Reply:
I'll just commit this since it's otherwise good to go 🙂